dev-tools/omdb/tests/successes.out

```diff
@@ -523,12 +523,14 @@ task: "instance_updater"
   currently executing: no
   last completed activation: <REDACTED ITERATIONS>, triggered by a periodic timer firing
     started at <REDACTED_TIMESTAMP> (<REDACTED DURATION>s ago) and ran for <REDACTED DURATION>ms
-    total instances in need of updates: 0
-      instances with Destroyed active VMMs: 0
-      instances with Failed active VMMs: 0
-      instances with terminated active migrations: 0
-    update sagas started: 0
-    update sagas completed successfully: 0
+    task explicitly disabled by config!
```

Comment:
I'm a little surprised to see this here - the `Default` policy should have `disabled` be false right?

Reply:
It's not disabled by default, but it _is_ disabled explicitly in the test config file. Since this task is intended as a backstop to ensure update sagas are always run *eventually*, but it's not actually the primary mechanism that triggers update sagas, we disable it when running Nexus' integration tests, to make sure the other mechanisms for starting update sagas work in the happy path. See: https://github.com/oxidecomputer/omicron/blob/a1534d783c20baaa5f57c4b65d219abe503ecaca/nexus/tests/config.test.toml#L127-L138

Reply:
:+1: